mmv1/products/compute/Disk.yaml

```diff
@@ -430,11 +430,20 @@ properties:
     name: 'provisionedIops'
     description: |
       Indicates how many IOPS must be provisioned for the disk.
-      Note: Update currently only supported by hyperdisk skus, allowing for an update of IOPS every 4 hours
+      Note: Update currently only supported by hyperdisk skus, allowing for an update of IOPS every 4 hours without deletion and recreations
     required: false
     default_from_api: true
     update_verb: :PATCH
     update_url: 'projects/{{project}}/zones/{{zone}}/disks/{{name}}?paths=provisionedIops'
+  - !ruby/object:Api::Type::Integer
+    name: 'provisionedThroughput'
+    description: |
+      Indicates how much Throughput must be provisioned for the disk.
```

Comment:
Should we include the full original description i.e. `Indicates how much throughput to provision for the disk. This sets the number of throughput mb per second that the disk can handle. Values must be between 1 and 7,124.`?

Reply:
So the values are harder to nail down and are based on the following  [link](https://cloud.google.com/compute/docs/disks/modify-hyperdisks#supported_values_for_volumes) I would say we just leave out the mins and maxes, at least until the documentation has had some improvements